README.EN.md

```diff
@@ -0,0 +1,549 @@
+# Django + React + Webpack
+
+Django is one of the most complete web frameworks available on the market. It manages everything from the database to the final HTML sent to the client. But with the introduction of the concept of [*single page apps*](wikipedia) to the front-end development environment, it has become usual to find systems using Django only to provide an [API](wikipedia) on the back-end that responds JSON data to be consumed by Javascript applications. This architecture that divides front and back-end into two separate code bases enables the independence between these sectors on the development process. It also makes it possible to have multiple client apps interacting with one API, ensuring the consistence of data and business rules and at the same time providing different user interfaces.
```

Comment:
consistence -> consistency
https://english.stackexchange.com/questions/40564/consistence-and-consistency